githubstudent.md

```diff
@@ -0,0 +1,4 @@
+# GitHub Student
+
+* Name: GitHub Student
```

Comment:
Yes this looks good! :+1: 

![kerm](https://cloud.githubusercontent.com/assets/2132216/10645054/500acbac-77f1-11e5-8d6a-48bd82e494ce.gif)